heroku_connect/db/models/base.py

```diff
@@ -60,6 +60,12 @@ class MyCustomObject(hc_models.HerokuConnectModel):
         on :attr:`settings.HEROKU_CONNECT_SCHEMA<.HerokuConnectAppConf.HEROKU_CONNECT_SCHEMA>`
         and :attr:`.sf_object_name`.
 
+    Note:
+
+        A model mixin must inherit from :class:`Meta.managed<django.db.models.Model>`
+        not `.HerokuConnectModel`. Only the final (not abstract) models should inherit
+        from `.HerokuConnectModel` otherwise build in fields will clash.
```

Comment:
I think you want the last sentence to be - `otherwise the built-in fields will clash`

Reply:
better?